app/apollo/type-defs.ts

```diff
@@ -59,10 +54,10 @@ export const typeDefs = `
     """
     A sample query to return details for the authenticated user currently logged in to our system
     """
-    currentUser: AuthenticatedUser
+    me: User
       @cypher(
         statement: """
-        MATCH (u:AuthenticatedUser {sub: $jwt.sub})
+        MATCH (u:User {sub: $jwt.sub})
```

Comment:
This should work...but does not. I have created an issue to report this with the current `@neo4j/graphql` alpha release at https://github.com/neo4j/graphql-tracker-temp/issues/4